crates/egui/src/widgets/plot/mod.rs

```diff
@@ -1650,6 +1650,10 @@ fn generate_marks(step_sizes: [f64; 3], bounds: (f64, f64)) -> Vec<GridMark> {
 /// Fill in all values between [min, max] which are a multiple of `step_size`
 fn fill_marks_between(out: &mut Vec<GridMark>, step_size: f64, (min, max): (f64, f64)) {
     assert!(max > min);
+    if step_size.abs() == 0.0 {
+        trace!("To small step_size");
```

Comment:
```suggestion
    if step_size == 0.0 {
        log::trace!("Ignoring step_size of zero");
```

… though can we still get a panic when `step_size` is very small, or `NaN`? What was the actual panic? Can we fix it at the source instead of adding a guard like this?